src/foam/core/types.js

```diff
@@ -435,7 +435,7 @@ foam.CLASS({
     [
       'adapt',
       function(_, v) {
-        return v.toLowerCase().trim();
+        return v || v === '' ? v : v.toLowerCase().trim();
```

Comment:
This doesn't make sense to me.

Reply:
If v is null, toLowerCase throws an uncaught type error